app/models/appointment.rb

```diff
@@ -67,10 +66,8 @@ def self.eligible_for_reminders(days_overdue: 3)
     overdue_by(days_overdue)
       .joins(:patient)
       .merge(Patient.contactable)
-      .left_joins(:communications)
-      .where(communications: {id: nil})
-      .left_joins(:appointment_reminders)
-      .where(appointment_reminders: {id: nil})
+      .left_joins(:notifications)
```

Comment:
We might have to make some changes to accommodate this. The reason I added the "where no communications" is to exclude all past appointments. Given that Notification is a new model, this will result in all past appointments receiving new notifications. This could also be fixed by changing
```
def self.overdue_by(number_of_days)
  overdue.where("scheduled_date <= ?", Date.current - number_of_days.days)
end
```
to only search for the scheduled date = date. But that would mean we might never send a reminder in some edge cases (notifications were turned off for a while, sidekiq dropped jobs).